### ISSUE TITLE
Security Vulnerability in Axios package

### ISSUE THREAD

**Issue by roninCode** [jjangga0214/hasura-cli#112]
I'm getting a dependabot error when using `"hasura-cli": "^2.33.4"`:

```
Axios Cross-Site Request Forgery Vulnerability

The latest possible version that can be installed is 0.21.4 because of the following conflicting dependency:

hasura-cli@2.35.1 requires axios@^0.21.1
```

I see that Axios is locked to versions prior to 1: https://github.com/jjangga0214/hasura-cli/blob/master/package.json#L93

It looks like: `The earliest fixed version is 1.6.0` - any way to update Axios to resolve this issue?

**Comment by jjangga0214** [comment]
Hi!
It's resolved by #110
Thanks for reporting :)

**Comment by some-user123** [comment]
Thanks for fixing this! However, a patched version seems not to be released yet. Any plans?
